roles/openshift_common/library/openshift_manage_iptables.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
```

Comment:
To keep our python code as consistent as possible, we've been adding the following line to the top of all of our python scripts. I know it doesn't solve all consistency problems, but it solves a lot of them.

# vim: expandtab:tabstop=4:shiftwidth=4